types/office-js-preview/index.d.ts

```diff
@@ -18491,14 +18547,266 @@ declare namespace Office {
+     *
+     * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose
+     * 
+     * @beta
+     */
+    export interface SensitivityLabelsCatalog {
+        /**
+         * Gets all the sensitivity labels enabled in Outlook.
+         * 
+         * @remarks
+         * [Api set: Mailbox preview]
+         * 
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: **read/write item**
+         *
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose
+         * 
+         * **Recommended**: To determine whether the sensitivty labels catalog is enabled in Outlook, call `getIsEnabledAsync` before using `getAsync`.
+         * 
+         * @param options - An object literal that contains one or more of the following properties:-
+         *        `asyncContext`: Developers can provide any object they wish to access in the callback function.
+         * @param callback - When the method completes, the function passed in the `callback` parameter is called with a single parameter, `asyncResult`,
+         *                 which is an `Office.AsyncResult` object. The available sensitivity labels and their properties are returned in the
+         *                 `asyncResult.value` property.
+         *  
+         * @beta
+         */
+        getAsync(options: Office.AsyncContextOptions, callback: (asyncResult: Office.AsyncResult<SensitivityLabelDetails[]>) => void): void;
+        /**
+         * Gets all the sensitivity labels enabled in Outlook.
+         * 
+         * @remarks
+         * [Api set: Mailbox preview]
+         * 
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/understanding-outlook-add-in-permissions | Minimum permission level}**: **read/write item**
+         *
+         * **{@link https://learn.microsoft.com/office/dev/add-ins/outlook/outlook-add-ins-overview#extension-points | Applicable Outlook mode}**: Compose
+         * 
+         * **Recommended**: To determine whether the sensitivty labels catalog is enabled in Outlook, call `getIsEnabledAsync` before using `getAsync`.
+         * 
+         * @param callback - When the method completes, the function passed in the `callback` parameter is called with a single parameter, `asyncResult`,
+         *                 which is an `Office.AsyncResult` object. The available sensitivity labels and their properties are returned in the
+         *                 `asyncResult.value` property.
+         *  
+         * @beta
+         */
+        getAsync(callback: (asyncResult: Office.AsyncResult<SensitivityLabelDetails[]>) => void): void;
+        /**
+         * Checks whether the sensitivity labels catalog is enabled in Outlook.
```

Comment:
Here and elsewhere, should this be singular?
```suggestion
         * Checks whether the sensitivity label catalog is enabled in Outlook.
```